planner/core/prepare_test.go

```diff
@@ -15,9 +15,13 @@ package core_test
 
 import (
 	. "github.com/pingcap/check"
+	"github.com/pingcap/tidb/infoschema"
+	"github.com/pingcap/tidb/metrics"
 	"github.com/pingcap/tidb/planner/core"
 	"github.com/pingcap/tidb/util/testkit"
 	"github.com/pingcap/tidb/util/testleak"
+	dto "github.com/prometheus/client_model/go"
+	"time"
```

Comment:
put this line to line 17 and leave a blank line behind it.

Reply:
Thank you for the detailed comment. I've updated the PR.